sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/Configs.java

```diff
@@ -110,6 +110,10 @@ public class Configs {
     private static final String REPLICA_ADDRESS_VALIDATION_ENABLED = "COSMOS.REPLICA_ADDRESS_VALIDATION_ENABLED";
     private static final boolean DEFAULT_REPLICA_ADDRESS_VALIDATION_ENABLED = true;
 
+    // whether to skip recording metric
+    private static final String IGNORE_METRIC_ENALBLED = "COSMOS.IGNORE_METRIC_ENABLED";
+    private static final boolean DEFAULT_IGNORE_METRIC_ENABLED = false;
```

Comment:
I think I would prefer something like COSMOS.ENABLE_LEGACY_METRICS - or COSMOS_ENABLE_VERBOSE_METRICS

Then we can disable by default - but have a fallback in case some customers use them and find them useful (don't think so, but better to have a plan-b)

Reply:
like the name VERBOSE_METRICS_ENABLED. Will update